app/apollo/resolvers/cluster.js

```diff
@@ -588,6 +599,9 @@ const clusterResolvers = {
           });
         }
 
+        // Allow graphQL plugins to retrieve more information
+        context.pluginContext = {registration: registration, uuid: cluster_id};
```

Comment:
Why entire registration object instead of:
```suggestion
        context.pluginContext = {name: registration.name, uuid: cluster_id};
```
?